controllers/logging/KafkaLogger.js

```diff
@@ -14,6 +14,7 @@ var kafkaLogger = {
         var saleInfo = JSON.parse(JSON.stringify(product));
         saleInfo["merchant_id"] = merchant_hash;
         saleInfo["timestamp"] = timeOfBuy;
+        saleInfo["price"] = product.price * product.amount;
```

Comment:
Do you think this is clear enough? I would rather use something like "order_sum"?